<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>corbeille</class>
 <widget class="QDialog" name="corbeille">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Corbeille</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/02-Recycle-icon.png</normaloff>:/icones/Icones/02-Recycle-icon.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../Res.qrc">
        <normaloff>:/icones/Icones/doss4.jpg</normaloff>:/icones/Icones/doss4.jpg</iconset>
      </attribute>
      <attribute name="title">
       <string>Dossier</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableView" name="dossierTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btdossierRestaurer">
           <property name="text">
            <string>Restaurer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/arrow_up_32.png</normaloff>:/icones/Icones/arrow_up_32.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btdossierSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../Res.qrc">
        <normaloff>:/icones/Icones/user1_32.png</normaloff>:/icones/Icones/user1_32.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Agent</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableView" name="agentTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btagentRestaurer">
           <property name="text">
            <string>Restaurer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/arrow_up_32.png</normaloff>:/icones/Icones/arrow_up_32.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btagentSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="icon">
       <iconset resource="../Res.qrc">
        <normaloff>:/icones/Icones/form.jpg</normaloff>:/icones/Icones/form.jpg</iconset>
      </attribute>
      <attribute name="title">
       <string>Formation</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QTableView" name="formationTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="btformationRestaurer">
           <property name="text">
            <string>Restaurer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/arrow_up_32.png</normaloff>:/icones/Icones/arrow_up_32.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btformationSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="icon">
       <iconset resource="../Res.qrc">
        <normaloff>:/icones/Icones/sanc.jpg</normaloff>:/icones/Icones/sanc.jpg</iconset>
      </attribute>
      <attribute name="title">
       <string>Sanction</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QTableView" name="sanctionTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="btsanctionRestaurer">
           <property name="text">
            <string>Restaurer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/arrow_up_32.png</normaloff>:/icones/Icones/arrow_up_32.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btsanctionSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="icon">
       <iconset resource="../Res.qrc">
        <normaloff>:/icones/Icones/trans.jpg</normaloff>:/icones/Icones/trans.jpg</iconset>
      </attribute>
      <attribute name="title">
       <string>Transporteur</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QTableView" name="transporteurTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPushButton" name="bttransporteurRestaurer">
           <property name="text">
            <string>Restaurer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/arrow_up_32.png</normaloff>:/icones/Icones/arrow_up_32.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttransporteurSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>corbeille</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
